src/s11/handlers/create_bearer_resp_handler.c

```diff
@@ -0,0 +1,140 @@
+        return -1;
+    }
+	
+	GtpV2MessageHeader gtpHeader;
+	gtpHeader.msgType =  GTP_CREATE_BEARER_RSP;
+	gtpHeader.sequenceNumber = g_s11_sequence;
+	gtpHeader.teidPresent = true;
+	gtpHeader.teid = cb_resp_msg->s11_sgw_cp_teid;
+    struct sockaddr_in sgw_ip = {0};
+    create_sock_addr(&sgw_ip, g_s11_cfg.egtp_def_port,
+                    cb_resp_msg->s11_sgw_cp_teid);
+
+	g_s11_sequence++;
+
+	CreateBearerResponseMsgData msgData;
+	memset(&msgData, 0, sizeof(CreateBearerResponseMsgData));
+
+	msgData.cause.causeValue = cb_resp_msg->cause;
+        
+	msgData.bearerContextsCount = cb_resp_msg->bearerCtxList.bearers_count;
+	for(int i = 0; i < msgData.bearerContextsCount; i++)
+	{
+	    msgData.bearerContexts[i].epsBearerId.epsBearerId = cb_resp_msg->bearerCtxList.bearerCtxt[i].eps_bearer_id;
+	    
+	    msgData.bearerContexts[i].cause.causeValue = cb_resp_msg->bearerCtxList.bearerCtxt[i].cause.data;
+
+	    msgData.bearerContexts[i].s1UEnodebFTeidIePresent = true;
+	    msgData.bearerContexts[i].s1UEnodebFTeid.ipv4present = 
+		    cb_resp_msg->bearerCtxList.bearerCtxt[i].s1u_enb_fteid.header.v4;
+	    msgData.bearerContexts[i].s1UEnodebFTeid.interfaceType = 
+		    cb_resp_msg->bearerCtxList.bearerCtxt[i].s1u_enb_fteid.header.iface_type;
+	    msgData.bearerContexts[i].s1UEnodebFTeid.teidGreKey =
+		    cb_resp_msg->bearerCtxList.bearerCtxt[i].s1u_enb_fteid.header.teid_gre;
+	    msgData.bearerContexts[i].s1UEnodebFTeid.ipV4Address.ipValue =
+		    cb_resp_msg->bearerCtxList.bearerCtxt[i].s1u_enb_fteid.ip.ipv4.s_addr;
+
+	    msgData.bearerContexts[i].s1USgwFTeidIePresent = true;
+            msgData.bearerContexts[i].s1USgwFTeid.ipv4present =
+                    cb_resp_msg->bearerCtxList.bearerCtxt[i].s1u_sgw_teid.header.v4;
+            msgData.bearerContexts[i].s1USgwFTeid.interfaceType =
+                    cb_resp_msg->bearerCtxList.bearerCtxt[i].s1u_sgw_teid.header.iface_type;
+            msgData.bearerContexts[i].s1USgwFTeid.teidGreKey =
+                    cb_resp_msg->bearerCtxList.bearerCtxt[i].s1u_sgw_teid.header.teid_gre;
+            msgData.bearerContexts[i].s1USgwFTeid.ipV4Address.ipValue =
+                    cb_resp_msg->bearerCtxList.bearerCtxt[i].s1u_sgw_teid.ip.ipv4.s_addr;
+
+	    if(cb_resp_msg->bearerCtxList.bearerCtxt[i].pco.pco_length > 0)
+	    {
+	        msgData.bearerContexts[i].protocolConfigurationOptionsIePresent = true;
+		memcpy(msgData.bearerContexts[i].protocolConfigurationOptions.pcoValue.values, 
+				cb_resp_msg->bearerCtxList.bearerCtxt[i].pco.pco_options, 
+				cb_resp_msg->bearerCtxList.bearerCtxt[i].pco.pco_length);
+	    }
+
+	}
+
+	if(cb_resp_msg->pco.pco_length > 0)
+	{
+	    msgData.protocolConfigurationOptionsIePresent = true;
+	    memcpy(msgData.protocolConfigurationOptions.pcoValue.values, cb_resp_msg->pco.pco_options,
+			    cb_resp_msg->pco.pco_length);
+	}
+
+	add_gtp_transaction(gtpHeader.sequenceNumber, cb_resp_msg->ue_idx);
```

Comment:
We shouldn't add to this map for CB Response.

Reply:
Removed.